lib/index.js

```diff
@@ -97,17 +110,27 @@ AdRoll.prototype.track = function(track) {
   del(customProps, 'orderId');
   del(customProps, 'id');
   del(customProps, 'sku');
-  if (!is.empty(customProps)) data.adroll_custom_data = customProps;
+  if (!is.empty(customProps)) {
```

Comment:
Why do we need to check for emptiness? `each` won't do anything on an empty set.